go/util/sizecache/size_cache.go

```diff
@@ -92,3 +92,15 @@ func (c *SizeCache) Add(key interface{}, size uint64, value interface{}) {
 		}
 	}
 }
+
+// Drop will remove the element associated with the given key from the cache.
+func (c *SizeCache) Drop(key interface{}) {
```

Comment:
Kinda bummer this class needs to use interface{} everywhere. Just an observation.